interface/forms_admin/forms_admin.php

```diff
@@ -61,150 +61,167 @@
-    stristr($fname, ".tar") || stristr($fname, ".zip") ||
-    stristr($fname, ".gz")))
-        $inDir[$i] = $fname;
+<div class="panel panel-default">
+    <div class="panel-heading"><?php xl('Unregistered','e');?></div>
+    <table class="table table-bordered" cellpadding=1 cellspacing=2>
+    <tr>
+        <td><?php xl('Name ','e'); ?></td>
+        <td><?php xl('Action ','e'); ?></td>
+        <td><?php xl('PHP ','e'); ?></td>
+        <td></td>
+    </tr>
+    <?php
+    $dpath = "$srcdir/../interface/forms/";
+    $dp = opendir($dpath);
+    $color="#eeeeee";
+    for ($i=0; false != ($fname = readdir($dp)); $i++)
+        if ($fname != "." && $fname != ".." && $fname != "CVS" && $fname != "LBF" &&
+        (is_dir($dpath.$fname) || stristr($fname, ".tar.gz") ||
+        stristr($fname, ".tar") || stristr($fname, ".zip") ||
+        stristr($fname, ".gz")))
+            $inDir[$i] = $fname;
 
-// ballards 11/05/2005 fixed bug in removing registered form from the list
-if ($bigdata != false)
-{
-    foreach ( $bigdata as $registry )
+    // ballards 11/05/2005 fixed bug in removing registered form from the list
+    if ($bigdata != false)
     {
-        $key = array_search($registry['directory'], $inDir) ;  /* returns integer or FALSE */
-        unset($inDir[$key]);
+        foreach ( $bigdata as $registry )
+        {
+            $key = array_search($registry['directory'], $inDir) ;  /* returns integer or FALSE */
+            unset($inDir[$key]);
+        }
     }
-}
 
-foreach ( $inDir as $fname )
-{
-        // added 8-2009 by BM - do not show the metric vitals form as option since deprecated
-    //  also added a toggle in globals.php in case user wants the option to add this deprecated form
-        if (($fname == "vitalsM") && ($GLOBALS['disable_deprecated_metrics_form'])) continue;   
-    
-    if (stristr($fname, ".tar.gz") || stristr($fname, ".tar") || stristr($fname, ".zip") || stristr($fname, ".gz"))
-        $phpState = "PHP compressed";
-    else
-        $phpState =  "PHP extracted";
+    foreach ( $inDir as $fname )
+    {
+            // added 8-2009 by BM - do not show the metric vitals form as option since deprecated
+        //  also added a toggle in globals.php in case user wants the option to add this deprecated form
+            if (($fname == "vitalsM") && ($GLOBALS['disable_deprecated_metrics_form'])) continue;   
+        
+        if (stristr($fname, ".tar.gz") || stristr($fname, ".tar") || stristr($fname, ".zip") || stristr($fname, ".gz"))
+            $phpState = "PHP compressed";
+        else
+            $phpState =  "PHP extracted";
+        ?>
+        <tr >
+            <td width="20%" class="buttoncol">
+                <?php
+                    $form_title_file = @file($GLOBALS['srcdir']."/../interface/forms/$fname/info.txt");
+                            if ($form_title_file)
+                                    $form_title = $form_title_file[0];
+                            else
```

Comment:
okay looks good

Reply:
Looks *sooo much better*.  I think it is a lot simpler to use.  Links to enable an install db work and everything.